docs/HowTo/Troubleshoot/Add-Validators-Without-Voting.md

```diff
@@ -0,0 +1,258 @@
+---
+description: How to add or remove validators without voting
+---
+
+# Add and remove validators without voting
+
+[QBFT](../Configure/Consensus-Protocols/QBFT.md) or [IBFT 2.0](../Configure/Consensus-Protocols/IBFT.md) network
+conditions might not allow voting to change validators.
+For example, if a majority of the current validators are no longer participating in the network, a vote to add or
+remove validators won't be successful.
+You can bypass voting and specify new validators in the genesis file.
+
+!!! warning
+
+    In most cases, use the voting method to add or remove validators.
+    Only use the without-voting method when voting isn't possible.
+    Using this method requires coordinating restarting all the nodes in order to pick up the configuration at the
+    correct block height.
+    Using this method also leaves the validator overrides permanently in your genesis configuration.
+
+To add or remove validators without voting:
+
+1. Stop all nodes in the network.
+1. In the genesis file, add the `transitions` configuration item where:
+
+    * `<BlockNumber>` is the upcoming block at which to change validators.
+    * `<ValidatorAddressX> ... <ValidatorAddressZ>` are strings representing the account addresses
+      of the validators after `<BlockNumber>`.
+
+    !!! example "Transitions object in the genesis file"
+
+        === "Syntax"
+
+            ```bash
+            {
+              "config": {
+                 ...
+                 "qbft": {  // or "ibft2"
+                   "blockperiodseconds": 2,
+                   "epochlength": 30000,
+                   "requesttimeoutseconds": 4
+                 },
+                 "transitions": {
+                   "qbft": [  // or "ibft2"
+                   {
+                     "block": <BlockNumber>,
+                     "validators": [
+                        <ValidatorAddressX>,
+                        ...
+                        <ValidatorAddressZ>
+                     ]
+                   }
+                   ]
+                 }
+              },
+              ...
+            }
+            ```
+
+        === "Example"
+
+            ```bash
+            {
+              "config": {
+                ...
+                "qbft": {  // or "ibft2"
```

Comment:
Could have another tab for ibft2? Similar to the example on https://besu.hyperledger.org/en/stable/Concepts/Consensus-Protocols/Overview-Consensus/

I think it's clear enough to read, but it would be nice if users can copy and paste this too. Unfortunately, this isn't valid JSON because comments aren't allowed :(

Appreciate it's a bit annoying having two sub-variants of the Syntax and Example tab...not sure what the best solution is here.

Reply:
Good point, fixed!